### PR TITLE
Links update

### PR DESCRIPTION
**Pull request by gocadimic**
Some minor changes in content/rewording and typo fixing were done at  8cddfd0d3ad98b9f8026ff583bfeeafb59b63299.

### CODE REVIEW

File: content/en/images.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Figures MUST be referenced using HTML
 * In a GitHub repository, each technical document should have its own "images" folder.
 * The location of the figure is controlled by the src parameter
 * See Creation of `.svg` Images
-    * Please follow these guidelines when creating a new image from scratch
+    * Please follow these [`guidelines`](https://docs.google.com/document/d/194oEy2GrUdLafICeKUxPVH8zX9okZd_D/edit?usp=sharing&ouid=112978134087833973902&rtpof=true&sd=true) when creating a new image from scratch.
```
Comment (jpradocueva) [inline content/en/images.md:34]: @gocadimic, the problem with that link is that the document is restricted to only SH staff. We need to download a PDF and upload it to the repository. Also, add a link inside the document to the link in GDrive. People won't be able to download it, but at least we know where the document is, say, six months from today.

**Review by jpradocueva — CHANGES_REQUESTED** [review]
@gocadimic I added inline a comment about the document. We cannot point to the document in the GDrive. It is restricted to SH staff only. We will need to download it, add a link to the doc in GDrive to remember where the master copy is and then create a pdf.

File: content/en/images.md (hunk 1 above)
Comment (gocadimic) [reply content/en/images.md:34]: Resolved in PR #26

**Review by jpradocueva — APPROVED** [review]
Thanks.

**Review by jpradocueva — APPROVED** [review]
There was a conflict that has to be resolved prior merging.

**Pull request merged**